translationsth.js

```diff
@@ -0,0 +1,43 @@
+{
+	"LOADING": "กำลังโหลด …",
+
+	"TODAY": "วันนี้",
+	"TOMORROW": "พรุ่งนี้",
+	"RUNNING": "สิ้นสุดใน",
+	"EMPTY": "ไม่มีกิจกรรมที่กำลังจะมาถึง",
+	"WEEK": "สัปดาห์muj {weekNumber}",
+
+	"N": "น",
+	"NNE": "น.ต.อ.น.",
+	"NE": "ต.อ.น.",
+	"ENE": "ต.อ.ต.อ.น.",
+	"E": "ต.อ.",
+	"ESE": "ต.อ.ต.อ.ต.",
+	"SE": "ต.อ.ต.",
+	"SSE": "ต.ต.อ.ต.",
+	"S": "ต.",
+	"SSW": "ต.ต.ต.ต.",
+	"SW": "ต.ต.ต.",
+	"WSW": "ต.ต.ต.ต.ต.",
+	"W": "ต.ต.",
+	"WNW": "ต.ต.ต.ต.น.",
+	"NW": "ต.ต.น.",
+	"NNW": "น.ต.ต.น.",
+
+	"FEELS": "รู้สึกเหมือน {DEGREE}",
+	"PRECIP": "ความแม่นยำ",
```

Comment:
this just got changed and split into two keys:


	"PRECIP_POP": "PoP",
	"PRECIP_AMOUNT": "Precipitation amount",

Reply:
Add already